### PR TITLE
Split Node docs into three sections

### PR DESCRIPTION
**Pull request by emmawillis**

### CODE REVIEW

File: docs/sdk/features/all-features.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The DevCycle React SDK is built upon the JavaScript SDK. For more details, view
 
 ## Server-Side SDK Usage
 
-### [• NodeJS SDK (server-side)](/sdk/server-side-sdks/node#getting-all-features)
+### [• NodeJS SDK (server-side)](/sdk/server-side-sdks/node/node-usage#getting-all-features)
```
Comment (suthar26) [inline docs/sdk/features/all-features.md:62]: we can drop the `(server-side)` looks weird
```suggestion
### [• NodeJS SDK](/sdk/server-side-sdks/node/node-usage#getting-all-features)
```

File: docs/sdk/features/all-variables.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ The DevCycle React SDK is built upon the JavaScript SDK. For more details, view
 
 ## Server-Side SDK Usage
 
-### [• NodeJS SDK (server-side)](/sdk/server-side-sdks/node#getting-all-variables)
+### [• NodeJS SDK (server-side)](/sdk/server-side-sdks/node/node-usage#getting-all-variables)
```
Comment (suthar26) [inline docs/sdk/features/all-variables.md:62]: same here looks weird when we have `(server-side)` only for this
```suggestion
### [• NodeJS SDK](/sdk/server-side-sdks/node/node-usage#getting-all-variables)
```

**Review by suthar26 — CHANGES_REQUESTED** [review]

File: docs/sdk/features/all-features.md (hunk 1 above)
Comment (emmawillis) [reply docs/sdk/features/all-features.md]: oh true, fixed!

File: docs/sdk/server-side-sdks/node/node-usage.md [hunk 3]
```diff
@@ -1,54 +1,13 @@
...
-```typescript
-import { initialize } from '@devcycle/nodejs-server-sdk'
-
-const dvcClient = await initialize('<DVC_SDK_SERVER_KEY>').onClientInitialized()
-```
-
-## Usage
-
-### Initialization Options
+## Initialization Options
```
Comment (chantalchung) [inline docs/sdk/server-side-sdks/node/node-usage.md:10]: should we put this with the Initialization steps in the Getting Started page?

Comment (emmawillis) [reply docs/sdk/server-side-sdks/node/node-usage.md:10]: good idea, i'll move it

**Review by chantalchung — APPROVED** [review]

File: docs/best-practices/serverless/serverless-cloud-functions.md [hunk 4]
```diff
@@ -110,7 +110,7 @@ Here is some more reading:
 First, you need to enable EdgeDB in the DevCycle Dashboard
 [https://docs.devcycle.com/home/feature-management/edgedb/edge-flags#setup](/home/feature-management/edgedb/edge-flags#setup)
 
-With the [DevCycle NodeJs SDK ](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details: ](/sdk/server-side-sdks/node#initialization-options))
+With the [DevCycle NodeJs SDK ](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details: ](/sdk/server-side-sdks/node/node-usage#initialization-options))
```
Comment (suthar26) [inline docs/best-practices/serverless/serverless-cloud-functions.md:113]:
```suggestion
With the [DevCycle NodeJs SDK ](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details: ](/sdk/server-side-sdks/node/node-gettingstarted#initialization-options))
```

Comment (chantalchung) [inline docs/best-practices/serverless/serverless-cloud-functions.md:113]: this one also needs to be updated to `/sdk/server-side-sdks/node/node-gettingstarted#initialization-options` !

**Review by suthar26 — CHANGES_REQUESTED** [review]

File: docs/best-practices/serverless/serverless-cloud-functions.md (hunk 4 above)
Comment (emmawillis) [reply docs/best-practices/serverless/serverless-cloud-functions.md]: oops - fixed, thanks!

**Review by suthar26 — APPROVED** [review]

**Pull request merged**